tests/lib/rules/no-invalid-v-on.js

```diff
@@ -58,7 +62,7 @@ tester.run('no-invalid-v-on', rule, {
     {
       filename: 'test.vue',
       code: '<template><div v-on:click></div></template>',
-      errors: ["'v-on' directives require that attribute value."]
+      errors: ["'v-on' directives require that attribute value or verb modifiers."]
```

Comment:
Even though `@` is a simple alias I think it would be good idea to add more valid test examples with `v-on`, like e.g.:
```
<template><div v-on:click.prevent></div></template>
<template><div v-on:click.native.stop></div></template>
```